meson dist fails

#### Expected behaviour
Tests like abicheck runs fine.

#### Actual behaviour
abicheck failed!
```
[0/1] Running all tests.
1/1 libmate-desktop symbols check        FAIL            1.16s   exit status 1
>>> ABI=/media/Data/Programme/Linux/Mate-Desktop/git-version/github-matedesktop/mate-desktop-dist/build/meson-private/dist-unpack/mate-desktop-1.27.2/libmate-desktop/mate-desktop.abi NM=/usr/bin/nm MALLOC_PERTURB_=94 LIB=/media/Data/Programme/Linux/Mate-Desktop/git-version/github-matedesktop/mate-desktop-dist/build/meson-private/dist-build/libmate-desktop/libmate-desktop-2.so.17.1.4 /media/Data/Programme/Linux/Mate-Desktop/git-version/github-matedesktop/mate-desktop-dist/build/meson-private/dist-unpack/mate-desktop-1.27.2/abi-check
―――――――――――――――――――――――――――――――――――――――――――――――――――― ✀  ―――――――――――――――――――――――――――――――――――――――――――――――――――――
New ABI detected - If intentional, update /media/Data/Programme/Linux/Mate-Desktop/git-version/github-matedesktop/mate-desktop-dist/build/meson-private/dist-unpack/mate-desktop-1.27.2/libmate-desktop/mate-desktop.abi.
mate_image_menu_item_get_type
―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――


Summary of Failures:

1/1 libmate-desktop symbols check FAIL            1.16s   exit status 1

Ok:                 0   
Expected Fail:      0   
Fail:               1   
Unexpected Pass:    0   
Skipped:            0   
Timeout:            0   

Full log written to /media/Data/Programme/Linux/Mate-Desktop/git-version/github-matedesktop/mate-desktop-dist/build/meson-private/dist-build/meson-logs/testlog.txt
FAILED: meson-internal__test 
/usr/bin/meson test --no-rebuild --print-errorlogs
ninja: build stopped: subcommand failed.
Running unit tests on the distribution package failed
Dist check build directory was /media/Data/Programme/Linux/Mate-Desktop/git-version/github-matedesktop/mate-desktop-dist/build/meson-private/dist-build
```


#### Steps to reproduce the behaviour
meson setup build
meson dist -C build

#### MATE general version
master 

#### Package version
master

#### Linux Distribution
Fedora

#### Link to bugreport of your Distribution (requirement)
not released

--- a/libmate-desktop/mate-desktop.abi
+++ b/libmate-desktop/mate-desktop.abi
@@ -148,6 +148,7 @@ mate_hsv_is_adjusting
 mate_hsv_new
 mate_hsv_set_color
 mate_hsv_set_metrics
+mate_image_menu_item_get_type
 mate_image_menu_item_new
 mate_image_menu_item_new_with_label
 mate_image_menu_item_new_with_mnemonic

